src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -138,7 +138,8 @@ static EmberAfStatus removeEntryFromGroupTable(EndpointId endpoint, GroupId grou
         EmberStatus status   = emberDeleteBinding(bindingIndex);
         if (status == EMBER_SUCCESS)
         {
-            uint8_t groupName[ZCL_GROUPS_CLUSTER_MAXIMUM_NAME_LENGTH + 1] = { 0 };
+            uint8_t groupNameBuffer[ZCL_GROUPS_CLUSTER_MAXIMUM_NAME_LENGTH] = { 0 };
+            chip::ByteSpan groupName(groupNameBuffer, sizeof(groupNameBuffer));
```

Comment:
This does not make sense to me.  This _used_ to pass in an empty string (first byte was 0).  Now it's passing in a span of size `ZCL_GROUPS_CLUSTER_MAXIMUM_NAME_LENGTH`.  This should just pass an empty (default-constructed) span, no?